conda_smithy/feedstock_content/.gitignore

```diff
@@ -9,5 +9,7 @@
 !/*/
 !/recipe/**
 !/.ci_support/**
+!/build_artifacts/**
```

Comment:
Would it be possible to add a comment for each of these lines. 


I am not very well versed in git ignore globbing syntax